.github/workflows/dev.yml

```diff
@@ -1,9 +1,9 @@
-name: Bygg og deploy master til dev
+name: Bygg og deploy til testmiljø
 
 on:
   push:
     branches:
-      - dev
+      - 'preprod/**'
```

Comment:
Hvorfor kaller vi det preprod og ikke dev, egentlig? Har sett begge deler rundt omkring, er det noen spesiell semantikk rundt begrepene?

Reply:
Det er to ulike navnestandarder. Preprod er litt tungt. Om det var det du tenkte på? Vi kan gjerne bytte for min del...

Reply:
Tror kaanskje at preprod er "gamle" måten...

Reply:
Ja, jeg tenker bare 'dev' er et finere navn, så hvis det bare er ny/gammel standard stemmer jeg for *dev*  😎